src/Instance/InstanceDetails/InstanceNewHolding/InstanceNewHolding.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import userEvent from '@testing-library/user-event';
+
+import '../../../../test/jest/__mock__';
+import renderWithIntl from '../../../../test/jest/helpers/renderWithIntl';
+
+import InstanceNewHolding from './InstanceNewHolding';
+
+const props = {
+  location: {},
+  instance: {},
+};
+
+const renderInstanceNewHolding = () => (
+  renderWithIntl(
+    <Router>
+      <InstanceNewHolding {...props} />
+    </Router>
+  )
+);
+
+describe('InstanceNewHolding', () => {
+  it('Should render and click the button', () => {
+    const { getByText, getByRole } = renderInstanceNewHolding();
+    expect(getByText).toBeDefined();
+    const HoldingLink = getByRole('button', { name: 'ui-inventory.addHoldings' });
+    userEvent.click(HoldingLink);
```

Comment:
Note: If you're going to fire an event, the next test should inspect how things change after the event. Here, it's unnecessary.